blog/2025-04-11-ten-years-of-conda-forge.md

```diff
@@ -0,0 +1,72 @@
+---
+authors:
+  - core
+---
+
+# Ten years of conda-forge!
+
+Today, 2025-04-11, marks the 10th anniversary of the conda-forge community.
+
+Join us in our Zulip instance and share how you got involved with conda-forge, how this community has helped you, or just to show appreciation to the thousands of volunteers that make this effort possible!
```

Comment:
I'd like to create a Zulip thread on Apr 11th and link it here before publishing it.